### DISCUSSION TITLE
How to get type file with custom build in typescript.

### DISCUSSION THREAD

**Discussion by bmcswrdd5977** [fabricjs/fabric.js#9191] [Q&A]
Like @types/fabric.
I need erasing,so build from http://fabricjs.com/build/.And how can i get this types file?

**Comment by asturur** [comment]
hi @bmcswrdd5977 the eraser brush as soon as it is converted will be its own import.
custom builds won't exist anymore.
There will be a base build with imports coming from 'fabric' and other extra code, still tested and supported coming eventually from 'fabric/erasing' ( i made the name up now ).
So that we can continue to support a not growing bundle but add features

The page fabricjs.com/build will go away with the release of 6.0
